.werft/installer-tests.ts

```diff
@@ -353,10 +354,16 @@ function runIntegrationTests() {
                 `werft log result -d "failed test" url "${testPhase.description}(Phase ${testPhase.phase}) failed. Please refer logs."`,
             );
 
-            sendFailureSlackAlert(testPhase.description, new Error("Integration test failed"), testPhase.slackhook)
+            const msg: string = slackAlerts.get(testPhase.slackhook) || ""
+            slackAlerts.set(testPhase.slackhook, msg + "\n" + testPhase.description)
```

Comment:
```suggestion
            slackAlerts.set(testPhase.slackhook, `${msg}\n${testPhase.description}`)
```
nit: modern JS for the win